src/Config/ObjectModel/Entity.cs

```diff
@@ -62,4 +73,36 @@ public record Entity
         Cache is not null &&
         Cache.Enabled is not null &&
         Cache.Enabled is true;
+
+    /// <summary>
+    /// Helper method to generate the linking entity name using the source and target entity names.
+    /// </summary>
+    /// <param name="source">Source entity name.</param>
+    /// <param name="target">Target entity name.</param>
+    /// <returns>Name of the linking entity.</returns>
+    public static string GenerateLinkingEntityName(string source, string target)
+    {
+        return LINKING_ENTITY_PREFIX + ENTITY_NAME_DELIMITER + source + ENTITY_NAME_DELIMITER + target;
```

Comment:
For Book-Author entities, will the linking entity be named `LinkingEntity$Book$Author`?

1. Is `$` a valid character in an entity name? 
2. Please consider adding an example in the method summary

Reply:
For entities exposed in the config, `$` is not allowed to be in the entity name. Hence using such a character helps us avoid any potential conflicts.

Reply:
Why not have this as a helper method in the `GraphqlNaming` or `GraphUtils` class? 

This can be re-used whenever the name of the linking entity needs to be determined

Reply:
The name of the linking entity does not seem to be a concern of GraphQLNaming but is our choice as to how we store it in the backend. I will move it to `GQLUtils`.